### ISSUE TITLE
Printer settings for stability 

### ISSUE THREAD

**Issue by TigerClaw33** [Zaribo/Zaribo-MK2#1]
Hi, 
which settings do I have to use for the parts?

At the moment I print with PETG, infill 100% and wall thickness/ top/bottom thickness 1.2mm. 

What are the best options for stability and for high temperatures?I think 20% infill and wall thickness of 1.2mm will be not enough for small parts ...

**Comment by murathanaraz** [comment]
Hi, 

%100 infill is an overkill. %20 infill will do the job perfectly. 0.20 layer height is the optimal resolution. 4 layers on bottom and top should do the trick. 

Please see the specs of your filament for temperature settings.

Cheers,
Murathan

**Comment by TigerClaw33** [comment]
Sometimes small parts are not stable enough at high temperatures and I thought it will be good to make it as strong as possible. Also for the screw holes maybe they will be more stable :)
The good think is I printed only the parts for the y axis with 100% infill. So for the extruder and the other axis maybe it would be to much with infill 100% also because the weight of the parts which have to be move.   

My next parts I will print with infill 20-25% and Grid instead of ZickZack -> (recommended from prusa3D), Top/bottom 4-5 and Wall Line Count 3 (5 would be to much for small parts because there will be not enough infill isn't it?)
Temperature is 220°C and for Fan Speed I am good with 20% for PETG. 50% destroys my structure ... 

I had many problems with extruder clicking and with 220°C, FanSpeed 20% and print speed 40mm/s I am very good ...

**Comment by murathanaraz** [comment]
Yes this is a kind of try and error thing. You will find the sweet spot. Variables are filaments and environmental temperatures. 

Cheers
